test/index.js

```diff
@@ -37,6 +37,37 @@ describe('Import type test', function() {
     expect(result.css.toString()).to.eql(EXPECTATION);
   });
 
+  it('escapes strings when escapeStrings option is set', function() {
```

Comment:
When the word "when" is used in the test description, it's usually a sign that part of it can be moved into a describe as grouping mechanism. It allows for adding multiple tests for the same condition. E.g.:

```javascript
describe('when escapeStrings options is set', function() {
  it('escapes string values', function() {
     ...
  });

  it('does something else', function() {
    ...
  });
});